.github/workflows/ci.yml

```diff
@@ -90,3 +90,13 @@ jobs:
         run: |
             cargo bench --bench riscv_machine
 
+  bench-nova-public-params:
```

Comment:
Why not also benchmark the provers themselves?

Reply:
Good question. I tried, criterion need 10 samples at least, one prover run on CI takes 4-6 minutes, which lead to at least 40 to 60 minutes to benchmark Prover. 
I have plan to move all CI tasks, including benchmark prover to self-host Github Action in near future.